examples/example-phx-1_3/test/example_phx_web/features/full_feature_test.exs

```diff
@@ -0,0 +1,123 @@
+defmodule ExamplePhxWeb.FullFeatureTest do
```

Comment:
Feature tests. There are more things to test, but this hits the areas most likely to smoke, so I think it's a pretty good start.